scheduling/dispatcher.rb

```diff
@@ -22,6 +22,10 @@ def scan
     ).order_by(:schedule).limit(idle_connections)
   end
 
+  def exception_to_hash(ex)
```

Comment:
`Clog.exception` may be useful for logging exceptions elsewhere as well

Reply:
I first thought, sure why not, but then I was thinking about it...how often do you want to log an exception *and* want to handle it in some material way?

Reply:
Yeah, you're correct. The backtrace is beneficial for unexpected errors, and we already catch them here.